diff --git a/default.nix b/default.nix
index 4e55c3c..50baf68 100644
--- a/default.nix
+++ b/default.nix
@@ -108,8 +108,8 @@ let
       runCommand (browser.name + "-wv") { version = browser.version; } ''
         mkdir -p $out
         cp -a ${browser}/* $out/
-        chmod u+w $out/libexec/chromium
-        cp -a ${widevine-cdm}/share/google/chrome/WidevineCdm $out/libexec/chromium/
+        chmod u+w $out/libexec/helium
+        cp -a ${widevine-cdm}/share/google/chrome/WidevineCdm $out/libexec/helium/
       ''
     else browser;
 
